Admin/FormAdmin.php

```diff
@@ -26,6 +26,7 @@
 class FormAdmin extends Admin
 {
     const LIST_ROUTE = 'sulu_form.list';
+    const LIST_ROUTE_DATA = 'sulu_form.list.data';
```

Comment:
The dot here is acting kind like a namespace. So I would name it `sulu_form.edit_form.data`, because it is only appearing in the edit form.